### PR TITLE
[dashboard] Hide 'stopping' & unpinned workspaces from 'Active'

### PR DESCRIPTION
**Pull request by jankeromnes**
Potential solution/workaround for https://github.com/gitpod-io/gitpod/issues/3879

How to test:
- Delete a running workspace
- It should immediately disappear from the workspaces list, just like when you delete an already-stopped workspace

(It actually gets hidden and not removed, so you'll still see the "stopping" workspace in `Filter: All`, but maybe that's okay for now.)

(EDIT: Another potential difference is, if you delete a running _pinned_ workspace, it will still continue to be displayed as "stopping" in `Filter: Active` for a while. A fix for that could be to immediately un-pin when you delete.)

### CODE REVIEW

File: .werft/values.dev.yaml
Comment (gtsiolis) [inline .werft/values.dev.yaml:38]: **reminder**: To remove this before merging! 😇

**Review by gtsiolis — APPROVED** [review]
Hey @jankeromnes! Thanks for giving this a spin.

Behavior for unpinned workspaces looks good. Should be fine to merge this and revert if the different behavior for deleting pinned workspaces causes any confusion.

File: .werft/values.dev.yaml
Comment (jankeromnes) [reply .werft/values.dev.yaml]: Indeed, split out into https://github.com/gitpod-io/gitpod/pull/4211

**Pull request merged**
